include/vcpkg/base/jsonreader.h

```diff
@@ -54,10 +55,10 @@ namespace vcpkg::Json
         void add_missing_field_error(StringView type, StringView key, StringView key_type);
         void add_expected_type_error(StringView expected_type);
         void add_extra_field_error(StringView type, StringView fields, StringView suggestion = {});
-        template<class... Args>
-        void add_generic_error(StringView type, Args&&... args)
+        template<class A1, class... Args>
+        void add_generic_error(StringView type, const A1& arg1, const Args&... args)
```

Comment:
It seems like this should accept a single LocalizedString rather than args... in the modern era ? (If that's too big a can of worms feel free to say that)

Reply:
That's too big a can of worms for now; for the future, absolutely true.